models/ecoli/sim/initial_conditions.py

```diff
@@ -85,7 +85,7 @@ def initialize_trna_charging(sim_data, states, variable_elongation):
 	# Estimate initial charging state
 	charging_params = get_charging_params(sim_data, variable_elongation=variable_elongation)
 	fraction_charged, *_ = calculate_trna_charging(synthetase_conc, uncharged_trna_conc,
-		charged_trna_conc, aa_conc, ribosome_conc, f, charging_params)
+		charged_trna_conc, aa_conc, ribosome_conc[0], f, charging_params)
```

Comment:
This change is surprising since `calculate_trna_charging(..., ribosome_conc, ...)` has the same docstring and it's use looks the same although restructured into `dcdt_jit()`.

Reply:
This was a funny discrepancy that was only exposed when I extracted `dcdt` from `calculate_trna_charging` for JIT compilation. In the highlighted change, `ribosome_conc` is an array with length one (derived from `ribosome_counts`, which gets its value by calling the [`counts`](https://github.com/CovertLab/wcEcoli/blob/c5217df64716ca447a6cdcb1bdb97df490e88e5c/wholecell/containers/unique_objects_container.py#L470-L472) method of a `UniqueObjectsContainer`). This leads to compilation errors because `dcdt_jit` does `min(v_rib, v_rib_max)`, where `v_rib` is derived from `ribosome_counts`.

It's pretty interesting that `min(np.array([1]), 2) == np.array([1])` and `min(np.array([2]), 1) == 1` in pure Python.